### PR TITLE
patch django for FIPS

### PR DESCRIPTION
**Pull request by mdellweg**
Not yet confirmed, this is the (only) patch we need.

But i don't quite remember if specifying the patch as an additional source in the spec is sufficient to get it applied.

### CODE REVIEW

**Review by bmbouter — APPROVED** [review]
This code looks good to me.

File: packages/python-django/python-django.spec [hunk 1]
```diff
@@ -10,6 +10,7 @@ Summary:        A high-level Python Web framework that encourages rapid developm
 License:        BSD
 URL:            https://www.djangoproject.com/
 Source0:        https://files.pythonhosted.org/packages/source/D/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+Source1:        0001-FIPS-Mark-use-of-MD5-not-security-relevant.patch
```
Comment (evgeni) [inline packages/python-django/python-django.spec]:
```suggestion
Patch0:        0001-FIPS-Mark-use-of-MD5-not-security-relevant.patch
```

Comment (evgeni) [reply packages/python-django/python-django.spec]: also needs to change

```
%autosetup -n %{pypi_name}-%{version}
```

to

```
%autosetup -n %{pypi_name}-%{version} -p1
```

(I guess -p1, based on the diff)

**Comment by evgeni** [comment]
@mdellweg yeah, now add `-p1` to the `%autosetup` call and it will build again

**Comment by mdellweg** [comment]
Thanks @evgeni 
Only when a test was read and turns back green, you know you fixed something. ;)

**Comment by evgeni** [comment]
Okay. This looks correct now, just needs a release bump and a changelog entry.

This should have tripped the CI, wonder why it didn't...

**Comment by mdellweg** [comment]
Right, where was my head today?

**Comment by evgeni** [comment]
My best guess would be Munich, but what do I know?! ;)

File: packages/python-django/python-django.spec [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
 Name:           python-%{srcname}
 Version:        2.2.16
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        A high-level Python Web framework that encourages rapid development and clean, pragmatic design
 
 License:        BSD
 URL:            https://www.djangoproject.com/
 Source0:        https://files.pythonhosted.org/packages/source/D/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+Patch0:        0001-FIPS-Mark-use-of-MD5-not-security-relevant.patch
```
Comment (evgeni) [inline packages/python-django/python-django.spec]:
```suggestion
Patch0:         0001-FIPS-Mark-use-of-MD5-not-security-relevant.patch
```

picky ;-)

**Comment by evgeni** [comment]
packaging wise this LGTM

File: packages/python-django/python-django.spec (hunk 2 above)
Comment (mdellweg) [reply packages/python-django/python-django.spec]: :roll_eyes:

**Comment by mdellweg** [comment]
I am quite confident now, this _is_ the only thing we need to patch.
Thenceforth I am putting it out of draft mode.

**Review by evgeni — APPROVED** [review]

**Pull request merged**